.gitignore

```diff
@@ -1,3 +1,6 @@
 /target
 **/*.rs.bk
 Cargo.lock
+
+# File made from tests
+create_file_then_persist.txt
```

Comment:
Please remove this, and modify the test to create the tempfile in a temp directory (outside the crate).

Reply:
make sense. ill do that. i did this because i dont know which approach to use.